README.md

```diff
@@ -858,6 +858,7 @@ Table of Contents
    * [Radmin VPN](https://www.radmin-vpn.com/) - Connect multiple computers together via a VPN enabling LAN-like networks. Unlimited peers. (Hamachi alternative)
    * [localhost.run](https://localhost.run/) — Instantly share your localhost environment! No download required. Run your app on port 8080 and then run this command and share the URL.
    * [ngrok.com](https://ngrok.com/) — Expose locally running servers over a tunnel to a public URL.
+   * [RudderStack](https://rudderstack.com/) - Open source, warehouse-first customer data pipeline and Segment-alternative. Collects and routes clickstream data and builds your customer data lake on your data warehouse. Hosted RudderStack is free up to 500k events per month with no time limit.
```

Comment:
nice thanks, please list it once only to keep the list managable

Reply:
Sorry about that. I accidentally left a duplicate next to Segment. Thought I removed that one.